### ISSUE TITLE
Offer Preferences Subject Groups

### ISSUE THREAD

**Issue by Revadike** [bartervg/barter.vg#41]
In addition to https://github.com/bartervg/barter.vg/issues/19, I'm proposing to add configurable columns for offer preferences. So, in addition to regular offers, users can choose to configure other groups of offers - like automated offers, offers by friends, offers by person x, etc - in a different way. This adds great flexibility and a good option to place certain settings, like opting in/out from certain type of offers. 
I imagine the user can add and remove these columns (except the default one) on demand and scroll horizontally, much like discord handles their role permissions, but in table style (it could be tabs or different pages too).

**Comment by Revadike** [comment]
tl;dr: Basically, this allows you to select which groups specific offer preferences apply to.

**Comment by Revadike** [comment]
Also, this should be added to https://github.com/bartervg/barter.vg/projects/3
